### PR TITLE
i18n(es): update `vercel`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updates `vercel`

#### Related issues & labels (optional)

- Closes # N/A
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 1]
```diff
@@ -220,7 +220,78 @@ export default defineConfig({
...
 
-### includeFiles
+### `isr`
+
+**Tipo:** <code>boolean |   VercelISRConfig</code><br/>
+**Disponible para:** Serverless
+<Since v="7.2.0" pkg="@astrojs/vercel" />
+**Por defecto**: `false`
+
+Le permite a tu proyecto ser desplegado como una función de [ISR (Regeneración Estática Incremental)](https://vercel.com/docs/incremental-static-regeneration), que almacena en caché tus páginas renderizadas bajo demanda de la misma manera que las páginas prerenderizadas después de la primera solicitud.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:230]:
```suggestion
Permite desplegar tu proyecto como una función de [ISR (Regeneración Estática Incremental)](https://vercel.com/docs/incremental-static-regeneration), que almacena en caché tus páginas renderizadas bajo demanda de la misma manera que las páginas prerenderizadas después de la primera solicitud.
```

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 2]
```diff
@@ -330,11 +403,13 @@ Puedes usar el middleware de Vercel Edge para interceptar una solicitud y rediri
 **¿Intentando reescribir?** Actualmente, reescribir una solicitud con middleware solo funciona para archivos estáticos.
 :::
 
-### Middleware de Vercel Edge con middleware de Astro
+### Running Astro middleware on Vercel Edge Functions
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:406]: falta traducir

File: src/content/docs/es/guides/integrations-guide/vercel.mdx [hunk 3]
```diff
@@ -330,11 +403,13 @@ Puedes usar el middleware de Vercel Edge para interceptar una solicitud y rediri
...
+
+El adaptador `@astrojs/vercel/serverless` puede crear una [función edge](https://vercel.com/docs/functions/edge-functions) a partir de un middleware de Astro en tu base de código. Cuando `edgeMiddleware` está habilitado, una función edge ejecutará tu código de middleware para todas las solicitudes, incluyendo activos estáticos, páginas prerrenderizadas y páginas renderizadas bajo demanda.
 
-El adaptador `@astrojs/vercel/serverless` puede crear automáticamente el middleware de Vercel Edge a partir de un middleware de Astro en tu código base.
+Para página renderizadas bajo demanda, el objeto `context.locals` se serializa usando JSON y se envía en un encabezado para la función sin servidor, que realiza la renderización. Como medida de seguridad, la función sin servidor se negará a servir solicitudes con una respuesta `403 Forbidden` a menos que provengan de la función edge generada.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/vercel.mdx:410]:
```suggestion
Para páginas renderizadas bajo demanda, el objeto `context.locals` se serializa usando JSON y se envía en un encabezado para la función serverless, que realiza la renderización. Como medida de seguridad, la función serverless se negará a servir solicitudes con una respuesta `403 Forbidden` a menos que provengan de la función edge generada.
```

**Review by Mardroide — APPROVED** [review]
LGTM ✅

**Pull request merged**